Disc/Networking/Helpers/Request.swift

```diff
@@ -115,3 +115,19 @@ func createRequestParameters(scopes scopes: [Scope] = [], provider: Provider, co
         + createRequestParameters(code: code)
         + createRequestParameters(redirectUri: redirectUri)
 }
+
+func createRequestParameters(clientId clientId: String, scopes: [Scope] = [], provider: Provider, token: String, secret: String? = .None) -> [String: String] {
+    return createRequestParameters(clientId: clientId)
+        + createRequestParameters(scopes: scopes)
+        + createRequestParameters(provider: provider)
+        + createRequestParameters(token: token)
+        + createRequestParameters(secret: secret)
+}
+
+func createRequestParameters(clientId clientId: String, scopes: [Scope] = [], provider: Provider, code: String, redirectUri: String) -> [String: String] {
+    return createRequestParameters(clientId: clientId)
+        + createRequestParameters(scopes: scopes)
+        + createRequestParameters(provider: provider)
+        + createRequestParameters(code: code)
+        + createRequestParameters(redirectUri: redirectUri)
+}
```

Comment:
> Thankfully, these aren't public so we can address later.

By that, I meant looking into an implementation that uses partial application / currying could wait. I still think we should leverage the existing functions to keep things in sync with `addIdentity`.